src/data/sql-quiz.ts

```diff
@@ -1,4 +1,41 @@
 const sqlQuiz = [
+  {
+    Question: "In SQL, which of the following is the correct syntax for creating a table?",  
+    Answer: "CREATE TABLE table_name (column1_name datatype, column2_name datatype)",
+    Distractor1: "CREATE TABLE table_name (column1_name, column2_name )",
+    Distractor2: "UPDATE TABLE table_name (column1_name datatype, column2_name datatype)",
+    Distractor3: "CREATE NEW TABLE table_name (column1_name datatype, column2_name datatype)",
+    Explanation: "In SQL tables are created by using the CREATE TABLE statement as well as specifying each columns datatype.",
+    Link: "https://www.freecodecamp.org/news/sql-create-table-statement-with-example-syntax/",
+  },
+  {
+    Question: "In SQL, what command is used to convert from one datatype to another?",
+    Answer: "CAST()",
+    Distractor1: "CHANGE()",
+    Distractor2: "TYPE()", 
+    Distractor3: "CONVERT()",
+    Explanation: "The CAST() function is used to convert a column in a table from one datatype to another.",  
+    Link: "https://www.freecodecamp.org/news/cast-a-function-in-sql-convert-char-to-int-sql-server-example/",
+  },
+  {
+    Question: "In SQL, which of the following is the correct syntax for inserting a new record(row) to a table?",
+    Answer: "INSERT INTO table_name (column1, column2, column3) VALUES (value1, value2, value3);", 
+    Distractor1: "ADD ROW table_name (column1, column2, column3) VALUES (value1, value2, value3);",
+    Distractor2: "ALTER table_name (value1, value2);",
+    Distractor3: "INSERT INTO table_name (column1, column2, column3) VALUES (value1, value2);",
```

Comment:
We want to try to keep the answers on the shorter side when possible

```suggestion
    Answer: "INSERT INTO table_name (column1, ...) VALUES (value1, ...);", 
    Distractor1: "ADD ROW table_name (column1, ...) VALUES (value1, ...);",
    Distractor2: "ALTER table_name (value1,...);",
    Distractor3: "INSERT INTO table_name (column1, ...) VALUES (value1,...);",
```